crates/goose/src/context_mgmt/mod.rs

```diff
@@ -31,13 +31,12 @@ struct SummarizeContext {
 /// # Returns
 /// * A tuple containing:
 ///   - `Conversation`: The compacted messages
-///   - `Vec<usize>`: Token counts for each message
-///   - `Option<ProviderUsage>`: Provider usage from summarization
```

Comment:
nice, I saw we didn't use this. There's some code on the frontend you could clean up with this too